stage1/README.md

```diff
@@ -195,6 +195,8 @@
 - Tasks:
 
   - ['Clean Code S1E1'](modules/clean-code/clean-code-s1e1.md)
+  - [coreJs-conditions-n-loops](https://github.com/rolling-scopes-school/coreJs-conditions-n-loops-tasks)
+  - [core-js-arrays](https://github.com/rolling-scopes-school/core-js-arrays)
```

Comment:
Possibly, for future task and repository names, it would be reasonable to decide on the writing style (`core-js`, or `coreJS`, or even `coreJs`), to make link labels consistent.